tests/unit/v1/test_async_collection.py

```diff
@@ -51,6 +51,9 @@ def test_query_method_matching(self):
         from google.cloud.firestore_v1.async_query import AsyncQuery
 
         query_methods = self._get_public_methods(AsyncQuery)
+        # Remove the one query-only method
```

Comment:
Maybe _copy is a private method? Is the intent for purely internal use or do you foresee users calling it?

Reply:
I suppose it could be private 🤔